store/src/java/com/zimbra/cs/util/Zimbra.java

```diff
@@ -330,7 +330,10 @@ private static synchronized void startup(boolean forMailboxd) throws ServiceExce
             }
 
             if (app.supports(ContactBackupThread.class.getName())) {
-                ContactBackupThread.startup();
+                int lastProcessedId = Config.getInt(Config.CONTACT_BACKUP_LAST_MAILBOX_ID, 0);
+                if (lastProcessedId > 0) {
```

Comment:
backup should not start automatically after server restart as now it's controlled by the utility. after server restart admin should start the backup as per the convenient time.

Reply:
this was discussed and decided while re-grooming.
If there is no list provided while starting contact backup, and server restarts before finishing, contact backup should start where it was stopped.